extension.json

```diff
@@ -27,21 +27,21 @@
 		}
 	},
 	"AutoloadNamespaces": {
-		"SimpleBatchUpload\\": "src/"
+		"MediaWiki\\Extension\\SimpleBatchUpload\\": "src/"
```

Comment:
While this is fine, beware that we often follow the `VendorName\\PackageName` pattern, so `\\ProfessionalWiki\\SimpleBatchUpload` in this case.